storage/client/queries/queries.go

```diff
@@ -421,10 +421,10 @@ const (
 			) AS eth_creation_tx,
 			creation_bytecode,
 			runtime_bytecode,
-			(
+			COALESCE((
 				SELECT gas_for_calling FROM chain.runtime_accounts ra
 				WHERE (ra.runtime = $1) AND (ra.address = $2::text)
-			) AS gas_for_calling,
+			), 0) AS gas_for_calling,
```

Comment:
unrelated tiny bugfix; it's possible to have a known contract with no associated known account, IIRC because we learned about the contract from sourcify